From: Sean Darcy <darcys22@gmail.com>
Date: Fri, 9 Jul 2021 15:29:06 +1000
Subject: Allow HF18 to be used earlier

The get earliest version returns HF 18.1 and this means that our blink
fees that were enabled in HF18 are no longer accepted. Push through to
allow the fees to be used 4000 blocks earlier (current height)
---
 src/wallet/wallet2.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/wallet/wallet2.cpp b/src/wallet/wallet2.cpp
index 03ca668..e1fb6d4 100644
--- a/src/wallet/wallet2.cpp
+++ b/src/wallet/wallet2.cpp
@@ -7783,7 +7783,7 @@ uint64_t wallet2::get_fee_percent(uint32_t priority, txtype type) const
       THROW_WALLET_EXCEPTION(error::invalid_priority);
 
     uint64_t burn_pct = 0;
-    if (use_fork_rules(network_version_18, 0))
+    if (use_fork_rules(network_version_18, 4000))
       burn_pct = BLINK_BURN_TX_FEE_PERCENT_V18;
     else
       THROW_WALLET_EXCEPTION(error::invalid_priority);
